packages/beacon-node/src/util/sszBytes.ts

```diff
@@ -38,6 +38,7 @@ export function getBlockRootFromAttestationSerialized(data: Uint8Array): BlockRo
 }
 
 export function getAttDataHashFromAttestationSerialized(data: Uint8Array): AttDataHash {
```

Comment:
If we are using base64 I would be extremely explicit everywhere. Same this function `getAttDataBase64FromAttestationSerialized` and when using it, name the variable `attDataBase64`

Reply:
I changed it accordingly 👍